packages/editor/src/components/preview-dropdown/index.js

```diff
@@ -62,33 +65,88 @@ export default function PreviewDropdown( { forceIsAutosaveable, disabled } ) {
 		desktop,
 	};
 
+	const isZoomOutExperiment =
+		!! window.__experimentalEnableZoomedOutPatternsTab;
+
+	const zoomIcon = () => {
+		if ( __unstableGetEditorMode() === 'zoom-out' ) {
+			return __( '50%' );
+		}
+		return __( '100%' );
+	};
+
+	const getIcon = () => {
+		if ( isZoomOutExperiment && deviceType === 'Desktop' ) {
+			return zoomIcon;
+		}
```

Comment:
Should it be the Desktop SVG + the %? Like `[desktop icon] 50%`?

Reply:
Or maybe the text can be _within_ the desktop icon? Like, the percentage is displayed on the screen if it's at 50%?

Reply:
I think to do that we would need new SVGs